docs/submitting-applications.md

```diff
@@ -114,12 +114,12 @@ run it with `--help`. Here are a few examples of common options:
   /path/to/examples.jar \
   1000
 
-# Run on a YARN cluster
+# Run on a YARN cluster in cluster deploy mode
```

Comment:
In this case, could you update the PR title and description about this YARN doc change?

Reply:
Sure one sec

Reply:
Done, please check, thanks @dongjoon-hyun